packages/b/box2d/patches/2.4.0/fix-build-scripts.patch

```diff
@@ -0,0 +1,38 @@
+From 0853481927429ad826d750e9a33332aef460b084 Mon Sep 17 00:00:00 2001
+From: PucklaMotzer09 <jonaas.pucher000000@gmail.com>
+Date: Wed, 2 Sep 2020 11:21:58 +0200
+Subject: [PATCH] fix build scripts
+
+---
+ build.bat | 4 ++--
+ build.sh  | 2 +-
+ 2 files changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/build.bat b/build.bat
+index 216561a..0c19b99 100644
+--- a/build.bat
++++ b/build.bat
+@@ -1,7 +1,7 @@
++@echo off
+ rem Use this batch file to build box2d for Visual Studio
+ rmdir /s /q build
+ mkdir build
+ cd build
+-cmake ..
++cmake %* ..
+ cmake --build .
+-start box2d.sln
+diff --git a/build.sh b/build.sh
+index 3433595..482d376 100644
+--- a/build.sh
++++ b/build.sh
+@@ -4,5 +4,5 @@
+ rm -rf build
+ mkdir build
+ cd build
+-cmake -DBOX2D_BUILD_DOCS=OFF ..
++cmake $@ ..
+ cmake --build .
```

Comment:
These two scripts are only a few lines, we can directly write them in on_install().

```lua
os.mkdir("build")
os.vrunv("cmake", {"--build", "."}, {curdir = "build"}) 
```

This is more common, there is no need to write a separate patch file.